tox.ini

```
@@ -2,11 +2,11 @@
 min_version = 4.0
 envlist =
     lint
-    py{39,310,311}-esmpy
+    py{39,310,31,312}-esmpy
```

```suggestion
    py{39,310,311,312}-esmpy
```
I don't think we support Python 3.1 ;)

---

Oof. Now that's a fatfinger typo.